docs/content.zh/release-notes/flink-1.16.md

```diff
@@ -0,0 +1,206 @@
+
+#### Support Retryable Lookup Join To Solve Delayed Updates Issue In External Systems
+
+##### [FLINK-28779](https://issues.apache.org/jira/browse/FLINK-28779)
+
+Adds retryable lookup join to support both async and sync lookups in order to solve the delayed updates issue in external systems.
+
+#### Make `AsyncDataStream.OutputMode` configurable for table module
+
+##### [FLINK-27622](https://issues.apache.org/jira/browse/FLINK-27622)
+
+It is recommend to set the new option 'table.exec.async-lookup.output-mode' to 'ALLOW_UNORDERED' when
+no stritctly output order is needed, this will yield significant performance gains on append-only streams
+
+#### Harden correctness for non-deterministic updates present in the changelog pipeline
+
+##### [FLINK-27849](https://issues.apache.org/jira/browse/FLINK-27849)
+
+For complex streaming jobs, now it's possible to detect and resolve potential correctness issues before running.
+
+### Connectors
+
+#### Move Elasticsearch connector to external connector repository
+
+##### [FLINK-26884](https://issues.apache.org/jira/browse/FLINK-26884)
+
+The Elasticsearch connector has been copied from the Flink repository to its own individual repository
+at https://github.com/apache/flink-connector-elasticsearch. For this release, identical Elasticsearch
+connector artifacts will be available from both repositories but with different versions. For example,
+the first releases will be `1.16.0` and the externally versioned and maintained artifact `3.0.0`.
+Developers are encouraged to move to the latter during this release cycle.
+
+#### Drop support for Hive versions 1.\*, 2.1.\* and 2.2.\*
+
+##### [FLINK-27044](https://issues.apache.org/jira/browse/FLINK-27044)
+
+Support for Hive 1.\*, 2.1.\* and 2.2.\* has been dropped from Flink. These Hive versions are no longer
+supported by the Hive community and therefore are also no longer supported by Flink.
+
+#### Hive sink report statistics to Hive metastore
+
+##### [FLINK-28883](https://issues.apache.org/jira/browse/FLINK-28883)
+
+In batch mode, Hive sink now will report statistics to Hive metastore by default for writting tables and partitions.
+This might be time-consuming when there are many written files. You can disable this feature by
+setting `table.exec.hive.sink.statistic-auto-gather.enable` to `false`.
+
+#### Remove a number of Pulsar cursor APIs
+
+##### [FLINK-27399](https://issues.apache.org/jira/browse/FLINK-27399)
+
+A number of breaking changes were made to the Pulsar Connector cursor APIs:
+
+- `CursorPosition#seekPosition()` has been removed.
+- `StartCursor#seekPosition()` has been removed.
+- `StopCursor#shouldStop` now returns a `StopCondition` instead of a boolean.
+
+#### Mark StreamingFileSink as deprecated
+
+##### [FLINK-27188](https://issues.apache.org/jira/browse/FLINK-27188)
+
+The StreamingFileSink has been deprecated in favor of the unified FileSink since Flink 1.12.
+This changed is reflected in the docs, but not yet in the codebase.
```

Comment:
```suggestion
```
@MartijnVisser This is properly deprecated in the code in https://github.com/apache/flink/commit/61a9d83abfe257892f961c6e2983a54b4e910622, correct?

Reply:
Yes, the code has been marked as deprecated too